packages/webchannel-wrapper/package.json

```diff
@@ -1,37 +1,41 @@
 {
   "name": "@firebase/webchannel-wrapper",
-  "version": "0.10.6",
+  "version": "0.10.7",
```

Comment:
Don't automatically bump the version, it will be done by the changesets tool during the release, based on your changeset file. (thick-spoons-check.md)